static/app/data/forms/projectGeneralSettings.tsx

```diff
@@ -149,7 +150,8 @@ export const fields: Record<string, Field> = {
     name: 'scrapeJavaScript',
     type: 'boolean',
     // if this is off for the organization, it cannot be enabled for the project
-    disabled: ({organization, name}) => !organization[name],
+    disabled: ({organization, project, name}) =>
+      !organization[name] || !hasEveryAccess(['project:write'], {organization, project}),
```

Comment:
Is this ‘hasEveryAccess’ condition implicit for all of the other settings on this page? I noticed that none of the other settings explicitly have this disabled condition despite being disabled by default.